models/endoscopic_inbody_classification/configs/metadata.json

```diff
@@ -1,8 +1,7 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.3.4",
+    "version": "0.3.3",
```

Comment:
I don't understand why change the version of this endoscopy back?
@wyli Do you know any context? I feel it's not our development proposal.

Thanks.

Reply:
I think it comes from the old version of the dev branch, there are same questions on ci files. Hi @myron , may need to sync with the latest dev branch, and solve the conflicts, thanks!

Reply:
okay, I've removed those changes (I'm not sure why/how it happened)